src/Common/tests/System/PlatformDetection.cs

```diff
@@ -31,8 +31,10 @@ public static partial class PlatformDetection
         public static bool IsNetBSD => RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"));
         public static bool IsOpenSUSE => IsDistroAndVersion("opensuse");
         public static bool IsUbuntu => IsDistroAndVersion("ubuntu");
-        public static bool IsWindowsNanoServer => (IsWindows && !File.Exists(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Windows), "regedit.exe")));
+        public static bool IsWindowsNanoServer => (IsWindows && IsNotWindowsIoTCore && !File.Exists(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Windows), "regedit.exe")));
```

Comment:
As ugly as it is, could you also add `IsNotWindowsNanoServerNorWindowsServerCore` ? This is so tests can annotate thus:
`[ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServerNorWindowsServerCore))]`

Although perhaps there's a better name, like `IsNotHeadlessWindows` or something? The point is they don't have GDI. (At least I assume that's why the Drawing tests fail on Nano and Server Core)